### PR TITLE
Create presidio-analyzer.json

### PR DESCRIPTION
**Pull request by Thorpe7**

### CODE REVIEW

**Comment by NPann** [comment]
@maxthorpe the image did not make it to dockerhub upon release.

**Comment by Thorpe7** [comment]
@NPann Image now appears on dockerhub.

File: gears/flywheel/presidio-analyzer.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    "flywheel": {
+      "suite": "PHI-Detection",
+      "show-job": true
+    },
+    "gear-builder": {
+      "category": "classifier",
+      "image": "flywheel/presidio-analyzer:0.1.0"
+    }
+  },
+  "description": "This gear utilizes Mircosoft's Presidio SDK which utilizes Natural Language Processing techiniques like Regex and NER to de-identify PII for subject and paitent data protection.",
```
Comment (NPann) [inline gears/flywheel/presidio-analyzer.json:22]: Can you make the following update:

* typo `paitent` -> `patient`
* explicit NER
* rephrase de-identify as "detect and report"

File: gears/flywheel/presidio-analyzer.json [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    "PYTHON_PIP_VERSION": "22.0.4",
+    "PYTHON_SETUPTOOLS_VERSION": "57.5.0",
+    "PYTHON_GET_PIP_URL": "https://github.com/pypa/get-pip/raw/1a96dc5acd0303c4700e02655aefd3bc68c78958/public/get-pip.py",
+    "PYTHON_GET_PIP_SHA256": "d1d09b0f9e745610657a528689ba3ea44a73bd19c60f4c954271b790c71c2653",
+    "FLYWHEEL": "/flywheel/v0"
+  },
+  "inputs": {
+    "input_file": {
+      "base": "file",
+      "description": "A text, csv, tsv, or json file"
```
Comment (NPann) [inline gears/flywheel/presidio-analyzer.json:37]: Nit: Can you add the correspond `enum` value for those file type ?

**Review by NPann — CHANGES_REQUESTED** [review]
lgtm

File: gears/flywheel/presidio-analyzer.json (hunk 1 above)
Comment (Thorpe7) [reply gears/flywheel/presidio-analyzer.json]: Changes implemented

File: gears/flywheel/presidio-analyzer.json (hunk 2 above)
Comment (Thorpe7) [reply gears/flywheel/presidio-analyzer.json]: enum values added

**Review by NPann — APPROVED** [review]

**Comment by NPann** [comment]
Thanks for addressing the comments @Thorpe7

**Pull request merged**